### PR TITLE
Home

### PR DESCRIPTION
**Pull request by gkiplangat**
Created a new branch with one file so far

### CODE REVIEW

**Review by glearnhub — CHANGES_REQUESTED** [review]
the code is okay, but you deleted the LICENSE file

File: LICENSE
Comment (gkiplangat) [inline LICENSE:1]: let me add it back

Comment (glearnhub) [reply LICENSE:1]: alright

**Review by gkiplangat — COMMENTED** [review]
i have added back the missing file LICENSE
image

**Review by glearnhub — APPROVED** [review]
i have seen on the updates

**Pull request merged**